chess/gaviota_native.py

```diff
@@ -0,0 +1,2436 @@
+import os
+from enum import Enum
+from lzma import LZMADecompressor
+
+MateResult = Enum ("MateResult", "WhiteToMate BlackToMate Draw Unknown")
+
+class ProbeResultType:
+    found = False
+    stm = MateResult.Unknown
+    error = ""
+    ply = 0
+    dtm = 0
+
+entries_per_block = 16 * 1024;
+currentStream = None
+currentFilename = ""
+validTables = [] # new List<string>(15);
+a8toa1 = {} #new Dictionary<int, int>();
+
+
+A1 = 0
```

Comment:
This happens to be the same square indexing scheme as the already defined `chess.A1`, ..., `chess.H8`.